client/my-sites/plans-v2/products-grid-i5/style.scss

```diff
@@ -29,3 +33,8 @@
 		grid-gap: 24px 16px;
 	}
 }
+
+.products-grid-i5__filter-bar {
+	height: 63px;
```

Comment:
Out of curiosity, why do we hardcode a height value? Is it necessary when we want the filter bar to stick?

Reply:
This is the wrapper div around the filter-bar, so when the inner filter-bar goes position fixed (and is suddenly removed from the document flow), this wrapper div stays static and maintains the height of the filter-bar, preventing the page from "jumping-up".  Another way to achieve this jumping prevention would be with css, adding margin, or padding, or translateY when the when the 'sticky' class is added. Either way, they are both hard-coded. I'm not sure if one is better than the other. I'm now wondering if there is a better way to achieve this. I'll look into it. If you are aware of a better solution here, please share 🙂